src/utils/format.ts

```diff
@@ -94,7 +94,7 @@ export const formatPSCForDisplay = (psc: PersonOfSignificantControl): PersonOfSi
 export const formatSecretaryList = (officers: ActiveOfficerDetails[]): any[] => {
   const secretaryList = new Array(0);
   for (const officer of officers) {
-    if (officer.role === "SECRETARY" && officer.isCorporate === false ) {
+    if (officer.role === "Secretary" && officer.isCorporate === false ) {
```

Comment:
why not use the enum OFFICER_ROLE? `officer.role === OFFICER_ROLE.SECRETARY`?

Reply:
ah completely forgot about it. will amend it